### ISSUE TITLE
SEO Checklist

### ISSUE THREAD

**Issue by wonderwomancode** [unlock-protocol/docs#61]
One time global
- [x] Review global metadata
- [x] Add robots file
- [x] Wire up Google Search Console
- [x] Optimize for Google Rich Results see https://search.google.com/test/rich-results

For every page
- [x] Review single page metadata https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-docs
- [x] Review for image metadata
- [x] Review pages for structured content inside each main page, note docusaurus already renders pages w/ semantic markup for sidebar <aside>, navbar <nav>, main page <main> 
- [x] Review pages for human readable links see https://docusaurus.io/docs/next/docs-introduction#document-id
- [x] Upon publication to final URL, run page through Twitter Card Validator and immediately update if share image or snippet is incorrect on desktop or mobile https://cards-dev.twitter.com/validator

**Comment by wonderwomancode** [comment]
Forgot to create branch in issue so linking here:
https://github.com/unlock-protocol/docs/tree/seo-updates

**Comment by wonderwomancode** [comment]
Added "Optimize for rich search results" to a new ticket #121  since it depends on #120

**Comment by wonderwomancode** [comment]
Review image metadata for individual pages can be done on a page-by-page basis as they're updated, as well as human readable links, as opposed to all at once now. Ticket opened to include those in the README on the repo #122
